crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -335,7 +335,7 @@ def addon_deployed?(addon)
         when "ceph"
           ::Node.find("roles:ceph-* AND ceph_config_environment:*").any?
         when "ha"
-          ::Node.find("pacemaker_founder:true AND pacemaker_config_environment:*").any?
+          ::Node.find("pacemaker_founder:* AND pacemaker_config_environment:*").any?
```

Comment:
Is this correct check? Isn't * true also for empty value?

Reply:
there should be no empty values on `node[:pacemaker][:founder]` either its false/true or the fqdn.

Doing a search on a non-existent attribute will return always empty:
```
irb(main):014:0> NodeObject.find("keystone_not_exists:*")
=> []

```

Reply:
Well, I think `::Node.find("pacemaker_config_environment:*")` gives the same result, right?

Reply:
it does, let me change it